.github/workflows/continuous-integration.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Run UI test
         if: always()
```

Comment:
@rafayet-monon This might not be related to this PR but can we remove this condition?

Reply:
@carryall If we remove this and when the UI test fails it won't run the next steps in the CI. It will exit.